packages/@ama-sdk/schematics/schematics/typescript/shell/index.ts

```diff
@@ -108,7 +116,8 @@ function ngGenerateTypescriptSDKFn(options: NgGenerateTypescriptSDKShellSchemati
 
   return chain([
     setupRule,
-    ...(options.skipInstall ? [] : [installRule])
+    ...(options.skipInstall ? [] : [installRule]),
+    ...(options.skipGit ? [] : [initGitRule])
```

Comment:
What happen if we create a sdk as a package in a monorepo ?
Because in this case I think we should not create a git repo as we are not at the root folder of the monorepo.

Reply:
If there is already a git repository, the [RepositoryInitializerTask](https://github.com/angular/angular-cli/blob/main/packages/angular_devkit/schematics/tasks/repo-init/executor.ts#L62) does nothing.

Reply:
But what happens if there is not (I don't know why but it could happen)

Reply:
If there is no git repository, then a new git repository is created where the sdk is generated, there is no detection of the monorepo. I think this is an acceptable behavior.